README.md

```diff
@@ -18,8 +18,38 @@ The PDS consists of:
 
 Review the [puppetlabs-puppet\_data\_service](https://github.com/puppetlabs/puppetlabs-puppet_data_service) module for detailed information on how to install, configure, and run the service.
 
+### Configuring the puppet_data_service module
+
+The `puppet_data_service` module will install [the PDS RPM](https://github.com/puppetlabs/puppet-data-service/releases) for you, so you don't have to worry about operationalizing the PDS service itself, dealing with DB setup, migrations, and so on, also it will install the PDS CLI as well.
+
+1. Add the [puppetlabs-puppet\_data\_service](https://github.com/puppetlabs/puppetlabs-puppet_data_service) module to your control repo
+   - Make sure to add the PDS hiera level in your control-repo's `hiera.yaml`
+2. Configure the two required roles
+   - The Database server
+     - Add a new Node Group from the PE Console.
+       ```
+       Parent name: PE Infrastructure
+       Group name: PDS Database
+       Environment: production
+       ```
+     - Add the class `puppet_data_service::database` to the PDS Database group created in the step above
+     - Add (pin) your PostgreSQL server certname in the Rules tab (it could be the primary server)
+     - Commit your changes
```

Comment:
Good call @timidri 

No, the PDS module will not install the DB if the DB server is not the primary PE.

I added[ instructions (Ln 48)](https://github.com/puppetlabs/puppet-data-service/pull/50/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R48) in case the node doesn't have a PostgreSQL installation yet.